doc/scenarios.md

```diff
@@ -0,0 +1,39 @@
+# Alternative Scenarios
+
+This is a short primer for how to use some of the parameters available in the witan.send model in order to run _"alternative scenarios"_.
+
+Each of the paramater keys (e.g. `:splice-ncy`) can be found, and their values defined, in the [config.edn](https://github.com/MastodonC/witan.send/blob/master/data/demo/config.edn).
+
+More information on how to define a scenario can be found here (TBD). 
+
+Here each scenario is simply defined in terms of the collection of parameters to use together, what arguments are expected and what the expected behaviuor is.
+
+### _“Ignore historic data before a specific calendar year for an age group”_
+
+* Parameters = `:filter-transitions-from` & `:splice-ncy`
+* Arguments = a calendar year and national curriculum year
```

Comment:
It's a little unclear to me how :splice-ncy works. Does this filter the data above, equal to and above, below etc? Perhaps give example args for your NCY 11 example below?